org.sn.myutils.pubsub/src/main/java/org/sn/myutils/pubsub/DistributedMessageServer.java

```diff
@@ -238,6 +284,7 @@ private static class TopicInfo {
             private final ReentrantLock lock = new ReentrantLock();
             private CreatePublisher createPublisher;
             private final List<SubscriberEndpoint> subscriberEndpoints = new ArrayList<>(); // unique by ClientMachine, subscriberName; sorted by ClientMachine, clientTimestamp
```

Comment:
Should we have 2 endpoints, one for pubsub and one for queue?
Why do we support one machine having 2 subscribers